source/Calamari.Common/Calamari.Common.csproj

```diff
@@ -50,7 +50,6 @@
     </ItemGroup>
 
     <ItemGroup>
-      <PackageReference Include="Octopus.CoreUtilities" Version="1.1.2" />
```

Comment:
Is this no longer needed?

Reply:
No, there was 1 extension method that was previously used, but it is no longer being used.